gravitee-apim-console-webui/src/shared/components/gio-diff/gio-diff.component.html

```diff
@@ -23,7 +23,7 @@
       (ngModelChange)="computeDiff()"
       aria-label="Output format button group"
     >
-      <mat-button-toggle value="row">Row</mat-button-toggle>
+      <mat-button-toggle value="row">Raw</mat-button-toggle>
```

Comment:
shouldn't we change the value also ?

Reply:
good catch ;)

Reply:
👍🏻  and also fix `line-by-lin` a few lines below